src/api/app/models/configuration.rb

```diff
@@ -50,6 +50,9 @@ def map_value(key, value)
     # Simple singleton implementation: Try to respond with the
     # the data from the first instance
     def method_missing(method_name, *args, &block)
+      unless first
```

Comment:
Why not `Configuration.first_or_create(name: 'private', title: 'Open Build Service', description: 'Private OBS Instance')`
http://apidock.com/rails/ActiveRecord/Relation/first_or_create